UPGRADE.md

```diff
@@ -2,7 +2,24 @@
 
 ## v0.4.\* -> v0.5.\*
 
-There are two major changes which occurred in this update:
+There are three major changes which occurred in this update:
+
+### Package Renamed
+
+This package is now called **Container Configurator**. It's not installed with
+the command:
+
+```
+composer install tomphp/container-configurator
+```
+
+For existing projects, you'll need to update your `composer.json`.
```

Comment:
@tomphp 

Should we suggest to use `composer` to update, instead of fiddling with `composer.json`?

```
composer remove tomphp/config-service-provider && composer require tomphp/container-configurator
```